src/FormModel.php

```diff
@@ -271,36 +272,44 @@ private function collectAttributes(): array
         $class = new ReflectionClass($this);
         $attributes = [];
 
+        $attributeLabels = $this->getAttributeLabels();
+        $attributeHints = $this->getAttributeHints();
+
         foreach ($class->getProperties() as $property) {
             if ($property->isStatic()) {
                 continue;
             }
 
-            /** @var ReflectionNamedType $type */
-            $type = $property->getType();
-            if ($type === null) {
+            /** @var ReflectionNamedType $propertyType */
+            $propertyType = $property->getType();
+            $propertyName = $property->getName();
+            if ($propertyType === null) {
                 throw new InvalidArgumentException(sprintf(
                     'You must specify the type hint for "%s" property in "%s" class.',
-                    $property->getName(),
+                    $propertyName,
                     $property->getDeclaringClass()->getName(),
                 ));
             }
 
-            $attributes[$property->getName()] = $type->getName();
+            $attributes[$propertyName] = new FormAttribute(
+                $propertyName,
+                $propertyType->getName(),
```

Comment:
Why?

Reply:
This will allow to do support nulled attributes in the future. For example, `?int $count`. On set empty string to attribute `count` value will be `null`.